zjit/src/cruby.rs

```diff
@@ -1218,6 +1218,14 @@ pub fn get_class_name(class: VALUE) -> String {
     }).unwrap_or_else(|| "Unknown".to_string())
 }
 
+pub fn class_has_safe_allocator(class: VALUE) -> bool {
```

Comment:
The meaning of "safe" seems ambiguous. Could you just call it `class_has_leaf_allocator`?